_articles/network.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: article
+title: Basic Networking Diagnosis
+description: Instructions on how to help diagnose networking issues.
+keywords:
+  - Support
+  - networking
+  - command line
+  - diagnosis
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: software-applications
+
+---
+
+Finding out why your system is having network issues can be a daunting task.  We'll explore some useful commands 
+
+[]{#anchor}A quick review of the TCP/IP model
```

Comment:
I'm not sure about the []{#anchor} format. Will this still make it an H1?

Reply:
ugh; that was from pandoc